### DISCUSSION TITLE
How to enable freeMode in a wrapper for React on web components?

### DISCUSSION THREAD

**Discussion by eno-tech0** [nolimits4web/swiper#6903] [Q&A] [answered]
I am using a wrapper for React on web components. When I add the setting freeMode: true nothing happens, here is an example: https://codesandbox.io/s/swiper-freemode-2nw993?file=/src/App.js
What is the correct way to use this setting with the React wrapper?

**Comment by nolimits4web** [comment, marked as answer]
```
freeMode: {enabled: true}
```

**Reply by eno-tech0** [reply]
Thanks, I thought it was supposed to work just with freeMode: true
